project/AkkaBuild.scala

```diff
@@ -213,103 +202,34 @@ object AkkaBuild extends Build {
     dependencies = Seq(persistence % "compile;provided->provided;test->test", testkit % "compile;test->test")
   )
 
-  lazy val streamAndHttp = Project(
```

Comment:
Hmm, not so sure, actually: adding the docs to this aggregate builds everything anyway, and running all HTTP tests is just two commands otherwise. Can this be solved differently? Perhaps adding a task that depends on those tests we want to execute?